packages/react-liveness/src/components/FaceLivenessDetector/LivenessCheck/LivenessCameraModule.tsx

```diff
@@ -221,6 +222,12 @@ export const LivenessCameraModule = (
     }
   }, [send, videoRef, isCameraReady, isMobileScreen]);
 
+  React.useEffect(() => {
+    if (isDetectFaceBeforeStart) {
+      clearOvalCanvas({ canvas: canvasRef.current! });
+    }
+  }, [send, videoRef, isDetectFaceBeforeStart]);
```

Comment:
What is the reasoning for inclusion of `send` and `videoRef` in the deps array? `videoRef` maintains a stable reference between renders, would assume the same for `send`

Reply:
whoops, good catch, was just a copy paste mistake, removed.